src/pretix/plugins/paypal2/templates/pretixplugins/paypal2/checkout_payment_confirm.html

```diff
@@ -2,10 +2,25 @@
 
 <p>
     {% if method == "wallet" %}
-        {% blocktrans trimmed %}
-            The total amount listed above will be withdrawn from your PayPal account after the
-            confirmation of your purchase.
-        {% endblocktrans %}
+        {% if url.url_name == "event.order.pay.confirm" %}
+            <div class="text-center" aria-hidden="true">
+                <span class="fa fa-info-circle fa-4x text-primary"></span>
+            </div>
+            <h2 class="text-center">
+                {% trans "Almost done…" %}
```

Comment:
AFAIK ellipsis directly after the word means the word is cut off, therefore I would add a space in-between.
```suggestion
                {% trans "Almost done …" %}
```